val.py

```diff
@@ -441,36 +453,88 @@ def run(
                 save_one_json(predn, jdict, path, class_map)  # append to COCO-JSON dictionary
             callbacks.run('on_val_image_end', pred, predn, path, names, im[si])
 
-            # TODO the following code contains a bug and will be fixed in PR #13
-            pred_clone[:, :4] = scale_boxes(im.shape[2:], pred_clone[:, :4],
-                                            im0[si].shape).round()
+            if skip_evaluation or save_blurred_image:
+                # TODO the following code contains a bug and will be fixed in PR #13
```

Comment:
is this TODO fixed?

Reply:
Still in PR #13, Want to dive some more in this code. Because of the None values when using cache